src/gui/optionsdialog.ui

```diff
@@ -231,8 +234,11 @@
                </item>
                <item>
                 <widget class="QCheckBox" name="checkAltRowColors">
+                 <property name="toolTip">
+                  <string>In the transfer list, every second row will have grey background</string>
```

Comment:
It's everywhere actually, except for the log and search widget.

Reply:
Addressed, let me know if the new tooltip reads right